import-service/package.json

```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "aws-sdk": "^2.792.0",
```

Comment:
Yo can move `aws-sdk` to **devDependencies** section. It will reduce your bundle size

Reply:
@mmarkelov
aws-sdk package is used inside lambda functions as part of main functionality. Could you please explain why it can be moved into devDependencies as it's required in production build?

Reply:
@svetlanagorchanina AWS runtime for Node has preinstalled AWS SDK, so there is no need to add it to prod build.
https://docs.aws.amazon.com/lambda/latest/dg/lambda-nodejs.html

Reply:
@mmarkelov got it, thanks!